### PR TITLE
Site contains basic functionality

### PR DESCRIPTION
**Pull request by rynkwn**

### CODE REVIEW

File: app/models/item.rb [hunk 1]
```diff
@@ -1,2 +1,8 @@
 class Item < ActiveRecord::Base
+    belongs_to :user
+    validates_presence_of :name
```
Comment (kevinchen93) [inline app/models/item.rb]: @rynkwn @BenDrews @modsoussi @ohjoseph 
The new Rails way is to do
```ruby
validates :price, presence: true, numericality: { greater_than_or_equal_to: 0 }
validates :name, presence: true
# etc
```

File: test/fixtures/users.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
```
Comment (kevinchen93) [inline test/fixtures/users.yml]: I've come to think that fixtures are evil. I normally `git rm test/fixtures/*.yml` since I tend to use [Factory Girl](https://github.com/thoughtbot/factory_girl) or [Fabrication](https://github.com/paulelliott/fabrication). When you write tests, I find it nice to initialize test objects and their associations in the actual test file, usually in the `setup` method. Additionally, when you run a migration and change the columns in a table, for some reason, I've found that you immediately update your factory initialization, whereas fixtures go stale without you knowing until some time later.

However, take what I'm saying with a grain of salt, as [fixtures have been known to be the choice for more complex applications](https://whatdoitest.com/getting-friendly-with-fixtures), e.g., the Rails core team. This is because as an app grows, it becomes tougher to test various edge cases, since you have to construct various scenarios with objects that have all sorts of crazy associations. To see this, just look at the dozens of lines of object initialization in [tickets_test](https://github.com/Niupiao/niupiao-rails/blob/master/test/integration/tickets_test.rb).
